projects/split-view/src/public-api.ts

```diff
@@ -0,0 +1,13 @@
+// Split view modules.
+export * from './modules/split-view/split-view.module';
+export {SkySplitViewComponent as λ1} from './modules/split-view/split-view.component';
```

Comment:
The "lambda" exports should include a comment, placed at the bottom of the file, see: https://github.com/blackbaud/skyux-core/blob/5.0.0-next/projects/core/src/public-api.ts#L70-L74